matrix-meetings-bot/src/service/MeetingService.ts

```diff
@@ -366,21 +362,15 @@ export class MeetingService {
     const oldMeeting = room.meeting;
     const newMeeting: IMeeting = { ...room.meeting };
 
-    // change data model if meeting is OX with non-empty rrules
-    const { start_time, end_time, calendar } = migrateMeetingTime(
+    // change data model if meeting is in old format
+    const calendar = migrateMeetingTime(
       meetingDetails,
       extractOxRrule(meetingDetails),
+      oldMeeting.calendar,
     );
 
-    newMeeting.calendar =
-      start_time === undefined && end_time === undefined
-        ? calendar ?? newMeeting.calendar
-        : undefined;
+    newMeeting.calendar = calendar ?? newMeeting.calendar;
```

Comment:
Is this still necessary? I think it is now always defined

```suggestion
    newMeeting.calendar = calendar;
```

Reply:
thanks, fixed that